deploy/examples/csi/rbd/storageclass-radosnamespace-test.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: replicapool
+  namespace: rook-ceph # namespace:cluster
+spec:
+  failureDomain: osd
+  replicated:
+    size: 1
```

Since removing the "test" suffix on the file, let's use replica 3
```suggestion
    size: 3
```